app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for human in people:
+        person = Person(human["name"], human["age"])
+        person_list.append(person)
+
+    for human in people:
+        for person in person_list:
+            if (person.name == human["name"]) \
+                    and ("wife" in human) \
+                    and (human["wife"] is not None):
+                person.wife = person.__class__.people[human["wife"]]
+            elif (person.name == human["name"]) \
+                    and ("husband" in human) \
+                    and (human["husband"] is not None):
```

Comment:
Could you avoid of `\`all over the project?

Reply:
Can you give me a hint? Because it doesn't work without `\`. And long expressions are also bad. Should I assign statements like `person.name == human["name"]` to variables and then put these variables into condition?

Reply:
@katevasilenko Please, look at my new commit. I've styled line breaks within parentheses. Is it the right direction?)